process.md

```diff
@@ -72,7 +78,7 @@ The overview should be a document (markdown, text, or PDF) consisting of the fol
 
     - For unsupported claims, explain why they are omitted. For example, this can be reasonable if it requires very long runs on clusters, or involves data sets that are proprietary. In this case, the authors should provide some mechanism by which claims can be tested by some other mechanism to establish the veracity of the approach, e.g., a smaller run or a different sample data set, with expected outputs.
```

Comment:
```suggestion
    - For unsupported claims, explain why they are omitted. For example, this can be reasonable if it requires very long runs on clusters, or involves data sets that are proprietary. In this case, the authors should provide some mechanism by which claims can be tested to establish the veracity of the approach, e.g., a smaller run or a different sample data set, with expected outputs.
```